### PR TITLE
Update encryption-office-365-certificate-chains.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about ISRG Root X1 as it replaces the DST root CA X3 which got expired last month.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6369

File: microsoft-365/compliance/encryption-office-365-certificate-chains.md [hunk 1]
```diff
@@ -141,19 +141,19 @@ Expand the root and intermediate sections below to see additional details about
...
-| **Subject Key Identifier** | c4:a7:b1:a4:7b:2c:71:fa:db:e1:4b:90:75:ff:c4:15:60:85:89:10 |
-| **Thumbprint (SHA-1)** | DAC9024F54D8F6DF94935FB1732638CA6AD77C13 |
-| **Thumbprint (SHA-256)** | 0687260331A72403D909F105E69BCF0D32E1BD2493FFC6D9206D11BCD6770739 |
-| **Pin (SHA-256)** | Vjs8r4z+80wjNcr1YKepWQboSIRi63WsWXhIMN+eWys= |
+### **ISRG Root X1*
+
+| **Subject** | C = US, O = Internet Security Research Group, CN = ISRG Root X1 |
+| --- | --- |
+| **Serial Number** | 82:10:cf:b0:d2:40:e3:59:44:63:e0:bb:63:82:8b:00 |
+| **Public Key Length** | RSA 4096 bit) |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/encryption-office-365-certificate-chains.md]:
```suggestion
| **Public Key Length** | RSA 4096 bit |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder.

**Comment by LauraWi** [comment]
Assigning to @kelleyvice-msft

**Comment by kelleyvice-msft** [comment]
I am assuming that this certificate has not been updated in the related .p7b file, so I will make that change and investigate whether other changes may be required due to other updates while I've been on leave.

**Pull request merged**